certificate/stse_certificate_prints.c

```diff
@@ -248,7 +248,7 @@ void stse_certificate_print_parsed_cert(stse_certificate_t *stse_certificate)
   //printf("Fields: %08X\n", stse_certificate->fields);
   //printFields(stse_certificate->fields);
 
-  printf("\n\r\t x509 Version: %ld", stse_certificate->x509Version + 1);
+  printf("\n\r\t x509 Version: %ld", (long)stse_certificate->x509Version + 1);
```

Comment:
If an effort is made to cast this variable, it would be preferable to use type macros to ensure platform portability.

You can include ```inttypes.h``` and replace all **long** cast following the example below.

```C
// EXISTING
printf("\n\r\t x509 Version: %ld", (long)stse_certificate->x509Version + 1);

// PROPOSAL
printf("\n\r\t x509 Version: %" PRId32, stse_certificate->x509Version + 1);
```